src/rocksdb_options.rs

```diff
@@ -565,6 +796,15 @@ impl Options {
     /// Measure IO stats in compactions and flushes, if true.
     ///
     /// Default: false
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use rocksdb::Options;
+    ///
+    /// let mut opts = Options::default();
+    /// opts.set_report_bg_io_stats(true);
+    /// ```
```

Comment:
Looks like this function was implemented recently. I can't find it in the 4.5 version of the CAPI: https://github.com/facebook/rocksdb/blob/4.5.fb/db/c.cc

Waiting for static linking to be merged would be the easiest solution to this.